lib/internal/navigator.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const {
+  ObjectDefineProperties,
+} = primordials;
+
+const {
+  kEnumerableProperty,
+} = require('internal/util');
+
+const {
+  getOSInformation,
+} = internalBinding('os');
+
+class Navigator {
+  get platform() {
+    switch (process.platform) {
+      case 'win32': return 'Win32';
+      case 'android': return 'Android';
+      default: return getOSInformation()[0];
```

Comment:
This is done because of spec compliance. Windows returns `Win32` on navigator.platform. Reference: https://html.spec.whatwg.org/multipage/system-state.html#dom-navigator-platform. 

Although, I'm fine with using `process.platform.`

Reply:
Are the values of `navigator.platform` standardized? It seems like we would get bug reports if they are different.

For example, on my machine `process.platform` is `'darwin'`, but `navigator.platform` is `MacIntel`.

Reply:
I pushed a fix for `darwin` where it should return `MacIntel` instead. This is the behavior of both Chrome and Safari for M1 devices.